libs/ui-watt/src/lib/components/chips/watt-chips.component.ts

```diff
@@ -0,0 +1,62 @@
+/**
+ * @license
+ * Copyright 2020 Energinet DataHub A/S
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License2");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+import {
+  Component,
+  EventEmitter,
+  Input,
+  Output,
+  ViewEncapsulation,
+} from '@angular/core';
+
+export interface WattChipsOption {
+  /** Text to display on the chip. */
+  label: string;
+  /** Value to emit when selected. */
+  value: string;
+}
+
+export type WattChipsSelection = string | null;
+
+/**
+ * Simple implementation of Material "Choice chips".
+ * @see https://material.io/components/chips#choice-chips
+ */
+@Component({
+  encapsulation: ViewEncapsulation.None,
+  selector: 'watt-chips',
+  styleUrls: ['./watt-chips.component.scss'],
+  templateUrl: './watt-chips.component.html',
```

Comment:
Suggestion: Add `changeDetection: ChangeDetectionStrategy.OnPush`

Here are some articles you can take a look at: 
https://angular.io/api/core/ChangeDetectionStrategy
https://www.thinktecture.com/en/angular/whats-the-hype-onpush/
https://blog.angular-university.io/onpush-change-detection-how-it-works/

Reply:
0f6e24b7bdcc3c1e7b1b4743df4c68b8d0284371